src/mock/createCanvasEvent.js

```diff
@@ -6,13 +6,18 @@
  * @example
  * interface CanvasRenderingContext2DEvent {
  *   type: string;
- *   transform: [number, number, number, number, number, number]; // the resulting current transform
+ *   transform?: [number, number, number, number, number, number];
+ *     // the resulting current transform
+ *     // when undefined, defaults to identity matrix [1, 0, 0, 1, 0, 0]
  *   props: {
  *     // if the event is a property was set, `event.props.value` would be set
  *     [propName: string]: any;
  *   };
  * }
  */
 export default function createCanvasEvent(type, transform, props) {
-  return { type, transform, props };
+  return transform
+  	? { type, transform, props }
+  	: { type, props }
```

Comment:
Again. I try to keep the interface exactly as it is to prevent v8 deoptimization. I don't think this is an appropriate change to jest-canvas-mock.

Reply:
hmm. make it optional?

Reply:
its even simpler: we can omit the `transform` field entirely, cos its 100% redundant
the transform state can always be derived from the past API calls to `ctx.setTransform` etc

so .. can we add an option to omit the `transform` field?
(of course in a zero-cost / unswitched way)